entries/h/harvard.edu.json

```diff
@@ -8,6 +8,12 @@
       "custom-software",
       "custom-hardware"
     ],
+    "custom-software": [
+      "Duo"
+    ],
+    "custom-hardware": [
+      "Yubico OTP"
```

Comment:
I assume "HarvardKey" are Yubikeys but I can't find any documentation that the OTP feature is the method being used.

Reply:
There was a reference to "Yubikey passcode" on [this page](https://harvard.service-now.com/ithelp?id=kb_article&sys_id=97c56d093778f68036fdd8d543990e78) and I assumed that was referring to OTP